--- origsrc/camlimages-2.2/configure.in	2004-10-01 11:13:18.000000000 -0500
+++ src/camlimages-2.2/configure.in	2008-01-09 03:03:05.296875000 -0600
@@ -276,8 +276,8 @@
 if test $SUPPORT_GIF = "true" ; then
   AC_MSG_CHECKING(-lungif/-lgif bug is fixed)
 
-  ORGLDFLAGS=$LDFLAGS
-  LDFLAGS="$LIBGIF $WITH_X"
+  ORGLIBS=$LIBS
+  LIBS="$LIBGIF $WITH_X"
   AC_TRY_RUN(
 #include <stdio.h>
 #include <gif_lib.h>
@@ -295,7 +295,7 @@
 -fwritable-strings. 
 *********************************************************************';
 AC_MSG_RESULT(with bug))
-LDFLAGS=$ORGLDFLAGS
+LIBS=$ORGLIBS
 fi
 
 dnl Checking -lpng ------------------------------------------------------------
--- origsrc/camlimages-2.2/tiff/tiffread.c	2004-09-21 16:56:44.000000000 -0500
+++ src/camlimages-2.2/tiff/tiffread.c	2008-01-09 02:57:52.515625000 -0600
@@ -28,11 +28,6 @@
 
 #include <tiffio.h>
 
-#undef int16
-#undef uint16
-#undef int32
-#undef uint32
-
 extern value *imglib_error;
 
 value open_tiff_file_for_read( name )
